ui/app/components/pages/confirm-transaction-switch/confirm-transaction-switch.component.js

```diff
@@ -32,24 +32,24 @@ export default class ConfirmTransactionSwitch extends Component {
       txData,
       methodData: { name },
       fetchingData,
-      isEtherTransaction,
+      // isEtherTransaction,
```

Comment:
Instead of commenting this out we should be able to remove it

Reply:
Awesome, it has been removed. I outlined why I removed this code [here](https://github.com/MetaMask/metamask-extension/issues/3425#issuecomment-427378933).